app/src/main/java/net/squanchy/speaker/SpeakerModule.java

```diff
@@ -0,0 +1,16 @@
+package net.squanchy.speaker;
+
+import net.squanchy.service.firebase.FirebaseSquanchyRepository;
+
+import dagger.Module;
+import dagger.Provides;
+
+@Module
+class SpeakerModule {
+
+    @Provides
+    SpeakerService speakerService(FirebaseSquanchyRepository repository) {
+        return new SpeakerService(repository);
+    }
+
+}
```

Comment:
Missing empty line... that's weird, it's set in preferences:

![image](https://cloud.githubusercontent.com/assets/153802/22891193/782828e2-f206-11e6-841d-4cc9c09107e6.png)